From a8f4c67ad3d5b12b87b49cb8f1b815dc3e90d5ca Mon Sep 17 00:00:00 2001
From: Arulselvan M <arulselvan.m@intel.com>
Date: Wed, 21 Nov 2012 13:56:49 +0530
Subject: [PATCH 029/170] [PORT FROM MAIN][Audio-JBMR1]: Enable WMA9 audio
 Decoding support

BZ: 50306

This patch contains changes in the libstagefright
to enable audio decoding capabilities

Change-Id: Ie4058be181b7bdfe1c22daa0a8c9139d8a1e403b
Signed-off-by: Arulselvan M <arulselvan.m@intel.com>
Reviewed-on: http://android.intel.com:8080/75141
Reviewed-by: lab_cactus <lab_cactus@intel.com>
Reviewed-by: Liang, Dan <dan.liang@intel.com>
Tested-by: Tong, BoX <box.tong@intel.com>
Reviewed-by: Tong, BoX <box.tong@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
---
 include/media/stagefright/MetaData.h |    7 +++++++
 1 files changed, 7 insertions(+), 0 deletions(-)

diff --git a/include/media/stagefright/MetaData.h b/include/media/stagefright/MetaData.h
index 01dca5e..6811394 100644
--- a/include/media/stagefright/MetaData.h
+++ b/include/media/stagefright/MetaData.h
@@ -157,6 +157,13 @@ enum {
     kKeyCryptoKey         = 'cryK',  // uint8_t[16]
     kKeyCryptoIV          = 'cryI',  // uint8_t[16]
     kKeyCryptoMode        = 'cryM',  // int32_t
+
+#ifdef USE_INTEL_ASF_EXTRACTOR
+    kKeyWmaBlockAlign     = 'blka',
+    kKeyWmaFormatTag      = 'frmt',
+    kKeyBitPerSample      = 'btps',
+#endif //ifdef USE_INTEL_ASF_EXTRACTOR
+
 };
 
 enum {
-- 
1.7.9

